lib/kennel/tasks.rb

```diff
@@ -8,6 +8,10 @@
 module Kennel
   module Tasks
     class << self
+      def kennel
+        @kennel ||= Kennel::Engine.new
+      end
```

Comment:
would it make more sense to have a `Kennel.engine` ?
seems a little strange that we have a global instance in the tasks ...

Reply:
I don't see why it would make more sense tbh. I'd prefer to have the library be more "clean", and the "default instance" (which is a form of state-sharing between rake tasks) handled by the rake tasks.